### ISSUE TITLE
[Feature] Allow running any tool on the file

### ISSUE THREAD

**Issue by jemag** [Allaman/kustomize.nvim#61]
It would be nice to be able to run any tool on a manifest file. So far `kubent` and `kubeconform` are supported, but there are plenty others that could be useful.

In my case, I think being able to run `trivy config {filename}` would be useful.

**Comment by Allaman** [comment]
Hello, thanks for your suggestion. This sounds like a useful feature! I will try to implement it

**Comment by Allaman** [comment]
Finally 😄 
Please tell me if it works 😆

**Comment by jemag** [comment]
hey thanks a lot for this and  sorry about the delay, been quite busy lately. Works great with the few tests I ran.

Only issue I encountered is sometimes I may build a large amount of manifests through kustomize (e.g.: argocd installation manifests), and it won't have time to complete and error out:
![image](https://github.com/user-attachments/assets/d27e8a23-f020-4e20-aa08-049a591c9981)

Would it be possible to edit the timeout either globally or for specific custom run commands?
